tests/classes/test_simplicialcomplex.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+import xgi
+from xgi.exception import XGIError
```

Comment:
We should probably try to add a couple of tests before merging :grin:

Reply:
I'll to add some in the coming days, if anyone feels inspired to write some too, feel free 😁

Reply:
Okay, 17 hours later.. done.